js/site.js

```diff
@@ -1,7 +1,44 @@
+class Score {
+    constructor() {
+        this.scoreValues = {};
```

Comment:
set properties to default values here?
at least that way, we know what they are.
(see comment below about `scoreValues` )